XU-TechnicalPaper.md

```diff
@@ -78,9 +78,15 @@ As with existing decentralized exchanges, no single point of failure (SPOF) can
 ---------------------
 The idea of payment channels is around for a while, even Satoshi Nakamoto already explored [*high-frequency transactions*](https://en.bitcoin.it/wiki/Payment_channels), then with bi-directional channels and the inception of the **Lightning Network** by Joseph Poon and Tadge Dryja, payment channels became more widely known. In short, payment channels enable instant, off-chain transfer of ownership of a digital token, like bitcoin, directly between two parties. Payment channels are opened and closed via one on-chain transactions each. Once a bidirectional payment channel is open, it uses unbroadcasted, fully valid signed transactions, which are sent back and forth between the two parties. These are held on each side of the channel, in our case on the exchange’s XU node, representing the balance of each party. These transactions will use a multi-signature address as their input (the funding address) and they will point at two different addresses for their output, each controlled by one party of the payment channel.
 
-Through their sophisticated setup, payment channels are trustless, because simply put, the said signed transactions are valid on the underlying chain and can be broadcasted at any point of time by either party. Also, payment channels allow trust-less routing of funds through intermediaries using a technology called [Hashed Timelock Contracts (HTLCs)](https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), thus a direct channel between two parties is not required and a path through other XU nodes can be used. This is described in detail in the [Lightning White Paper](https://lightning.network/lightning-network-paper.pdf). This is useful, since one payment channel to one other well-connected XU node would be enough to reach the rest of the network and also because setting up payment channels can be compared to the effort of transfering coins into a sidechain: coins have to be ‘locked’ into a payment channel via a native on-chain transaction.
+Through their sophisticated setup, even though off-chain, payment channels are trustless. This is because simply put, the said signed transactions are valid on the underlying chain and can be broadcasted at any point of time by either party. Also, payment channels allow trust-less routing of funds through intermediaries using a technology called [Hashed Timelock Contracts (HTLCs)](https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), thus a direct channel between two parties is not required and a path through other XU nodes can be used. This is described in detail in the [Lightning White Paper](https://lightning.network/lightning-network-paper.pdf). This is useful, since one payment channel to one other well-connected XU node would be enough to reach the rest of the network and also because setting up payment channels can be compared to the effort of transfering coins into a sidechain: coins have to be ‘locked’ into a payment channel via a native on-chain transaction.
```

Comment:
> Through their sophisticated setup, payment channels are trustless even though not all transactions are included in the blockchain. This is because intermediary transactions are signed and can be broadcast at any time by either party. Payment channels also allow trustless routing of funds through intermediaries using a technology called [Hashed Timelock Contracts (HTLCs)](https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), thus a direct channel between two parties is not required and a path through other XU nodes can be used. This is described in detail in the [Lightning White Paper](https://lightning.network/lightning-network-paper.pdf). This is useful since even a single payment channel to a well-connected XU node could be enough to reach the rest of the network Setting up payment channels can be compared to the act of transferring coins into a sidechain - coins have to be committed to a payment channel via a native on-chain transaction.

Note that I changed the word "locked" at the end to "committed" since locking implies that they are inaccessible, whereas in reality funds can be taken out of LN channels almost immediately with cooperating nodes and after a matter of days worst case.